tests/system/providers/google/cloud/dataprep/example_dataprep.py

```diff
@@ -0,0 +1,175 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that shows how to use Google Dataprep.
+"""
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.providers.google.cloud.operators.dataprep import (
+    DataprepCopyFlowOperator,
+    DataprepDeleteFlowOperator,
+    DataprepGetJobGroupOperator,
+    DataprepGetJobsForJobGroupOperator,
+    DataprepRunFlowOperator,
+    DataprepRunJobGroupOperator,
+)
+from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
+from airflow.providers.google.cloud.sensors.dataprep import DataprepJobGroupIsFinishedSensor
+from airflow.utils.trigger_rule import TriggerRule
+
+ENV_ID = os.environ.get('SYSTEM_TESTS_ENV_ID')
+DAG_ID = "example_dataprep"
+
+GCP_PROJECT_ID = os.environ.get('GCP_PROJECT_ID')
+GCS_BUCKET_NAME = f"dataprep-bucket-heorhi-{DAG_ID}-{ENV_ID}"
+GCS_BUCKET_PATH = f"gs://{GCS_BUCKET_NAME}/task_results/"
+
+FLOW_ID = os.environ.get('FLOW_ID', 1)
+RECIPE_ID = os.environ.get('RECIPE_ID')
+RECIPE_NAME = os.environ.get('RECIPE_NAME')
+WRITE_SETTINGS = (
+    {
+        "writesettings": [
+            {
+                "path": GCS_BUCKET_PATH,
+                "action": "create",
+                "format": "csv",
+            }
+        ],
+    },
+)
+
+with models.DAG(
+    DAG_ID,
+    schedule_interval="@once",
+    start_date=datetime(2021, 1, 1),  # Override to match your needs
+    catchup=False,
+    tags=['example', 'dataprep'],
+    render_template_as_native_obj=True,
+) as dag:
+    # [START how_to_gcs_create_bucket_operator]
```

Comment:
It's covered already by gcs examples, so we don't need START/END how_to_gcs_create_bucket_operator markers here.

Reply:
Removed